### ISSUE TITLE
nothing gets generated

### ISSUE THREAD

**Issue by thorizer** [vegardit/prisma-generator-nestjs-dto#49]
I'm using Prisma 3.7.0
`npx prisma migrate dev --name init` but no dto or entity gets generateed

```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator nestjsDto {
provider = "prisma-generator-nestjs-dto"
output = "../src"
}

model User {
  id           String         @id(map: "user_pk") @db.Uuid
  username     String         @unique(map: "user_username_uindex") @db.VarChar(255)
  email        String         @unique(map: "user_email_uindex") @db.VarChar(255)
  first_name   String         @db.VarChar(255)
  last_name    String         @db.VarChar(255)
  password     String         @db.VarChar(255)
  phone        String         @db.VarChar(255)
  phone_code   String?        @db.VarChar(50)
  last_login   DateTime?      @db.Date
  created_at   DateTime?      @default(now())
  updated_at   DateTime?      @updatedAt
  role         Role           @default(USER)
  article      Article[]
  user_address User_address[]
}

model User_address {
  id            String    @id(map: "user_address_pk") @db.Uuid
  user_id       String    @db.Uuid
  address_line1 String    @db.VarChar(255)
  address_line2 String    @db.VarChar(255)
  city          String    @db.VarChar(255)
  postal_code   String    @db.VarChar(255)
  country       String    @db.VarChar(255)
  created_at    DateTime? @default(now())
  updated_at    DateTime? @updatedAt
  gender        Gender
  user          User      @relation(fields: [user_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "user_id_fk")
}

model Article {
  id          String    @id(map: "article_pk") @db.Uuid
  author_id   String    @db.Uuid
  articleName String    @db.VarChar(255)
  isPublished Boolean   @default(false)
  created_at  DateTime? @default(now())
  updated_at  DateTime? @updatedAt
  user        User      @relation(fields: [author_id], references: [id], onDelete: NoAction, onUpdate: NoAction, map: "author_id_fk")
}

enum Gender {
  man
  woman
}

enum Role {
  USER
  ADMIN
  SUPERADMIN
}
```

**Comment by benkroeger** [comment]
have you tried `prisma generate` instead?

https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client

**Comment by thorizer** [comment]
@benkroeger I did. still no DTO

![Code_nBpY69pZri](https://user-images.githubusercontent.com/13587810/149296176-353d4553-1b84-4e11-a552-7526bde1649a.png)

**Comment by benkroeger** [comment]
please share repo with minimal reproduction

**Comment by thorizer** [comment]
@benkroeger  I've just made a minimal reproduction : [ GitHub repo](https://github.com/thorizer/nprodige)

**Comment by benkroeger** [comment]
this generator is a companion to the `prisma-client-js` generator. If you don't add that to your schema, noting will be generated.

```prisma
generator client {
  provider = "prisma-client-js"
}
```

**Comment by thorizer** [comment]
@benkroeger I did try that, generation gets stuck until i force exit 
![Code_QLZMZbGLLf](https://user-images.githubusercontent.com/13587810/149373333-2c92bf37-89fc-49be-81ab-7a6a1897e9d8.png)

**Comment by benkroeger** [comment]
this is related to https://github.com/prisma/prisma/issues/11126. The generator uses the latest `3.x` `@prisma/sdk` - so in this case, it installs `3.8.0` which tries to import `fs-jetpack`. Unfortunately, `fs-jetpack` is listed as `devDependency` in `@prisma/sdk` - and thus is missing.

to fix this until `@prisma/sdk@3.9.0` is released, please run

```sh
npm install --save-dev fs-jetpack
```

in your project. After that, I could successfully generate resources in your minimal reproduction

**Comment by thorizer** [comment]
@benkroeger, it works like a charm now, thank you.
